docs/build/setting-up/set-up-development-env.md

```diff
@@ -49,22 +49,23 @@ fluence dep un @fluencelabs/aqua-lib
 
 You can also edit `fluence.yaml` directly and run `fluence dep i` after that
 
-For advanced users: Aqua compiler currently relies on [npm](https://www.npmjs.com/) under the hood, to serve as it's package manager. This can change in the future. But currently, if you have any problems please refer to [npm documentation](https://docs.npmjs.com/). Please keep in mind that you don't need to separately install `npm` on your computer, because a specific version of npm is already included inside Fluence CLI itself
+For advanced users: The Aqua compiler currently relies on [npm](https://www.npmjs.com/) under the hood, to serve as its package manager, which may change in the future. For the time being, if you encounter any issues or problems please refer to [npm documentation](https://docs.npmjs.com/). Keep in mind that you don't need to install `npm` as Fluence CLI installs and manages its own version.
 
 ### Rust
 
-Fluence CLI will attempt to install Rust for you on Linux and macOS. If you are on Windows or CLI is unable to install Rust, please refer to the [official Rust documentation for installation instructions](https://www.rust-lang.org/tools/install)
+Fluence CLI will attempt to install Rust for you on Linux and macOS. If you are on Windows or the Fluence CLI is unable to install Rust, please refer to the [official Rust documentation for installation instructions](https://www.rust-lang.org/tools/install)
 
-Fluence CLI will also automatically install the required Rust toolchain and add `wasm32-wasi` target
+Fluence CLI will also automatically install the required Rust toolchain and add the `wasm32-wasi` target.
 
 In case of any problems related to Rust, here's a quick troubleshoot list:
 - Check you don't have 2 Rust installations (ie via rustup and brew on macOS)
 - Install wasm32-wasi manually: `rustup target add wasm32-wasi` & `rustup target add wasm32-wasi --toolchain nightly`
 - `rustup update`
+- Just reinstall rust completely from scratch
 
 ### Marine and Mrepl
 
-You will need these to build and test services written in Rust. They will be lazily downloaded from GitHub releases when they are needed. But you can also force the download by running
+In order to create and compile your Rust code to wasi and test the resulting Wasm modules, you need the Marine CLI and Marine REPL. Both components are wrapped by Fluence CLI and by default downloaded from the respective GitHub releases lazily when needed. Instead of the default lazy install, you can force the download by running
```

Comment:
```suggestion
In order to create and compile your Rust code to wasi and test the resulting Wasm modules, you need the Marine CLI and Marine REPL. Both components are wrapped by Fluence CLI and by default downloaded from the respective GitHub releases lazily. Instead of the default lazy install, you can force the download by running
```